src/main/groovy/com/athaydes/spockframework/report/internal/MultiReportCreator.groovy

```diff
@@ -0,0 +1,55 @@
+package com.athaydes.spockframework.report.internal
+
+import com.athaydes.spockframework.report.IReportCreator
+
+
+/**
+ * Container for multiple IReportCreators
+ */
+class MultiReportCreator implements IReportCreator {
+    private final List<IReportCreator> reportCreators = []
+
+    MultiReportCreator(List<IReportCreator> reportCreators) {
+        this.reportCreators.addAll(reportCreators)
+    }
+
+    @Override
+    void createReportFor(SpecData data) {
+        reportCreators.each {it.createReportFor(data)}
+    }
+
+    @Override
+    void setOutputDir(String path) {
+        reportCreators.each {it.setOutputDir(path)}
+    }
+
+    @Override
+    void setHideEmptyBlocks(boolean hide) {
+        reportCreators.each {it.setHideEmptyBlocks(hide)}
+    }
+
+    @Override
+    void setShowCodeBlocks(boolean show) {
+        reportCreators.each {it.setShowCodeBlocks(show)}
+    }
+
+    @Override
+    void setTestSourceRoots(String roots) {
+        reportCreators.each {it.setTestSourceRoots(roots)}
+    }
+
+    @Override
+    void setProjectName(String projectName) {
+        reportCreators.each {it.setProjectName(projectName)}
+    }
+
+    @Override
+    void setProjectVersion(String projectVersion) {
+        reportCreators.each {it.setProjectVersion(projectVersion)}
```

Comment:
The setters of this class are never supposed to be called because each report creator is already configured before you create an instance of this class... so I would suggest to just throw a `UnsupportedOperationException` to avoid writing code later that accidentally sets stuff on a report creator of this class as that would most likely be a mistake.

Reply:
Ah yes - makes sense - I've checked in a change to do it, thanks

Reply:
One last question, did you remember to trim the class names, so that adding whitespace between commas does not cause errors?

Reply:
Good point - I should be able to add that easily enough

Reply:
Ok it now handles rogue spaces - see what you think